client/llb/source.go

```diff
@@ -455,7 +454,7 @@ func Differ(t DiffType, required bool) LocalOption {
 	})
 }
 
-func OCILayout(contentStoreID string, dig digest.Digest, opts ...OCILayoutOption) State {
+func OCILayout(ref string, opts ...OCILayoutOption) State {
```

Comment:
Is there a need to change this? The old one makes it clear what arguments are expected. The new one requires a lot of guessing and should do a bunch of validation as only a very specific ref format is allowed.